docker-compose/_posts/2017-05-12-using-private-docker-images.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Using Private Docker Images
+short: Using Private Docker Images
+category: docker-compose
+order: 3
+development: true
+---
+
+Many teams use private Docker registries to host their images. To use your private images, refer to them in your Compose files or Dockerfiles and add your registry credentials to Runnable.
+
+---
+
+## Using Images in Your Files
+
+Use images in a Compose file by referring to them with the `image` configuration option:
```

Comment:
I would call it the "…`image` feature:`, as that's what the Compose docs calls them.